salt/modules/mount.py

```diff
@@ -867,6 +882,9 @@ def set_fstab(
                 line = salt.utils.stringutils.to_unicode(line)
                 try:
                     if criteria.match(line):
+                        log.debug(
+                            "Checking ( %s ) against current line ( %s )", entry, line
```

Comment:
If someone has credentials in their fstab file this would log the password I believe.

Reply:
removed